toolchain/check/context.h

```diff
@@ -365,11 +365,21 @@ class Context {
   // Returns whether the current position in the current block is reachable.
   auto is_current_position_reachable() -> bool;
 
-  // Returns the type ID for a constant of type `type`.
+  // Returns the type ID for a constant is a type value, i.e. it is a value of
+  // type `TypeType`.
+  //
+  // Facet values are also types, but can not be passed here. They should be
+  // converted to a type through an `as type` conversion, that is, to a value of
+  // type `TypeType`.
```

Comment:
Facets are not types, unless they're of type `type`.

Reply:
Hm, right.. but they are of "type value" [typishness](https://docs.google.com/presentation/d/1WJtP2yEBTKcKjvfYxnGJnN-sGmL4rCa9Xc7FWZcg0I8/edit#slide=id.p), but yes okay I see what you mean, thanks.

Reply:
Done, maybe the comments are feeling too explainy now, but they seem helpful to me, see what you think